src/ontodia/customization/props.ts

```diff
@@ -78,6 +79,7 @@ export interface LinkLabelStyle {
             fill?: string;
             'stroke'?: string;
             'stroke-width'?: number;
+            text?: LocalizedString[];
```

Comment:
According to our idea, we want to give the user ability to set those labels that he wants, depending on the link properties.